### PR TITLE
add profile diff and list subcommand.

### PR DESCRIPTION
**Pull request by morvencao**
add profile diff and list subcommand.

### CODE REVIEW

File: cmd/mesh/profile-list.go [hunk 1]
```diff
@@ -32,7 +34,16 @@ func profileListCmd(rootArgs *rootArgs) *cobra.Command {
 
 }
 
+// profileList list all the buildin profiles.
 func profileList(args *rootArgs) {
 	checkLogsOrExit(args)
-	fmt.Println("This command is not yet implemented.")
+	profiles := helm.ListBuildinProfiles()
+	if len(profiles) == 0 {
+		fmt.Println("No profile available.")
```
Comment (ostromart) [inline cmd/mesh/profile-list.go]: s/profile/profiles/

File: pkg/helm/vfs_renderer.go [hunk 2]
```diff
@@ -145,3 +145,14 @@ func stripPrefix(path, prefix string) string {
 	pv := strings.Split(path, "/")
 	return strings.Join(pv[pl:], "/")
 }
+
+// list all the buildin profiles.
+func ListBuildinProfiles() []string {
```
Comment (ostromart) [inline pkg/helm/vfs_renderer.go]: ProfileNames is map[string]bool for fast existence check, so values will always be true, you don't have to check for it.
You could make this a more general function (maybe in util) called StringBoolMapToSlice to make the code more reusable.

File: pkg/helm/vfs_renderer.go [hunk 3]
```diff
@@ -145,3 +145,14 @@ func stripPrefix(path, prefix string) string {
 	pv := strings.Split(path, "/")
 	return strings.Join(pv[pl:], "/")
 }
+
+// list all the buildin profiles.
```
Comment (ostromart) [inline pkg/helm/vfs_renderer.go]: s/buildin/builtin/

File: cmd/mesh/profile-list.go (hunk 1 above)
Comment (morvencao) [reply cmd/mesh/profile-list.go]: done

File: pkg/helm/vfs_renderer.go (hunk 2 above)
Comment (morvencao) [reply pkg/helm/vfs_renderer.go]: done

File: pkg/helm/vfs_renderer.go (hunk 3 above)
Comment (morvencao) [reply pkg/helm/vfs_renderer.go]: done

**Pull request merged**